modules/apigee/variables.tf

```diff
@@ -66,11 +66,13 @@ variable "instances" {
   type = map(object({
     name                          = optional(string)
     display_name                  = optional(string)
+    name                          = optional(string)
```

Comment:
Duplicate

Reply:
Sorry, must have got duplicated in a merge. I was working on this simultaneously.